docs/docs/tutorial/jsonnet.mdx

```diff
@@ -286,12 +286,22 @@ clusters just work.
 This allows us to make sure that you will never accidentally apply to the wrong
 cluster.
 
-> **Note**: Tanka won't create the namespace for you. Either include it in
-> Jsonnet, or with the following command:
+> **Note**: Tanka won't create the namespace for you -- you need to include it in
+> Jsonnet using:
+> ```json
+> {
+>   my_namespace: $.core.v1.namespace.new('my-namespace'),
```

Comment:
At this time ksonnet lib is not introduced yet .. so this snippet is not obvious and won’t work. 

We at least need a link I guess

Reply:
Slightly updated, PTAL.